packages/angular_devkit/build_angular/test/browser/svg_spec_large.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { runTargetSpec } from '@angular-devkit/architect/testing';
+import { join, normalize, virtualFs } from '@angular-devkit/core';
+import { tap } from 'rxjs/operators';
+import { browserTargetSpec, host, outputPath } from '../utils';
+
+describe('Browser Builder allow svg', () => {
+
+  beforeEach(done => host.initialize().toPromise().then(done, done.fail));
+  afterEach(done => host.restore().toPromise().then(done, done.fail));
+
+  it('can use .component.svg as templates (uses raw-loader and does not copy .svg to dist)',
```

Comment:
We’d need to add addition tests, such to test that with AOT, resources get inlined as expected.

Reply:
I can give it a try. Making the change to the webpack.config took me about 10 minutes, figuring out how tests work in angular-cli and writing those tests took about  5 hours, so I might need some help to do that.

Reply:
I am here to answer any questions.

Reply:
For the test with a ```@Component``` and aot, the test in styles_spec_large.ts only tests that the build is successful, but does not test if the aot compiler output is correct. How would I do that here? 

I could try to figure out what the correct compiler output would look like and do a contains-test for that in main.js. Would that be the correct way to test this? However, such a test would probably break when ivy is enabled, as the compiler output would be different.

Reply:
You can try to see if the actual template contain is in the bundle such as a string. That should be enough to determine if it worked or not.